tasks/90_alternc-custom.yml

```diff
@@ -16,6 +16,72 @@
 #   comment: 'You can specify a logo for the menu, example /images/my_logo.png .'
 # (...)
 
+## As empirically detected in https://github.com/UdelaRInterior/ansible-AlternC/issues/12
+## AlternC needs a first admin login into panel for the AlternC variables set to be efective. 
+- name: "Hook: admin web access to AlternC panel just installed"
+  uri:
+    url: '{{ alternc_desktop_protocol }}://{{ alternc_debconf_desktopname }}/login.php'
+    method: POST
+    body_format: form-urlencoded
+    body: 
+      username: admin
+      password: '{{ alternc_admin_pass }}'
+  register: alternc_panel_login
+  become: no
+  delegate_to: localhost
+  ignore_errors: yes
+
```

Comment:
This method to set AlternC configuration is not as good as if AlternC-API would no be broken, but it's better than setting values directly into the database, as do most of our tasks. It avoids missing in the role any task outside the database that would trigger the API, such as refreshing, or triggering the cron refresh of a configuration file. It could be generalized to other tasks, and may be needed for some, as we discovered for DNS slave IP authorization data, see: #19.